src/__tests__/cms-page/[...slug].test.tsx

```diff
@@ -133,6 +141,24 @@ describe.skip('CMS Page', () => {
     ).toBeInTheDocument();
     expect(screen.queryByText(page.content)).toBeInTheDocument();
   });
+
+  it('renders with sidebar layout when sidebar has content', async () => {
```

Comment:
I think these integration tests should be done for CmsPage.tsx component now that our testing library is not working.